controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Enforcer.java

```diff
@@ -0,0 +1,25 @@
+package com.yahoo.vespa.hosted.controller.api.role;
+
+import com.yahoo.config.provision.SystemName;
+
+import java.net.URI;
+
+/**
+ * Checks whether {@link Role}s have the required {@link Privilege}s to perform {@link Action}s on given {@link java.net.URI}s.
+ *
+ * @author jonmv
+ */
+public class Enforcer {
```

Comment:
![enforcer](https://images-na.ssl-images-amazon.com/images/I/91q8Dck0LxL._RI_SX300_.jpg)

Reply:
Hehe. Not 100% sure about the name.